### PR TITLE
add field nationality on donation entity

### PR DESCRIPTION
**Pull request by Raulnet**

### CODE REVIEW

File: src/Donation/DonationRequest.php [hunk 1]
```diff
@@ -126,6 +132,7 @@ public function __construct(
         $this->country = 'FR';
         $this->setAmount($amount);
         $this->duration = $duration;
+        $this->nationality = 'FR';
```
Comment (ottaviano) [inline src/Donation/DonationRequest.php]: `AppBundle\Address\Address::FRANCE`

File: src/Entity/Donation.php [hunk 2]
```diff
@@ -114,7 +119,8 @@ public function __construct(
         string $emailAddress,
         PostAddress $postAddress,
         string $clientIp,
-        int $duration = PayboxPaymentSubscription::NONE
+        int $duration = PayboxPaymentSubscription::NONE,
+        string $nationality = 'FR'
```
Comment (ottaviano) [inline src/Entity/Donation.php]: `AppBundle\Address\Address::FRANCE`

File: src/Donation/DonationRequest.php (hunk 1 above)
Comment (Raulnet) [reply src/Donation/DonationRequest.php]: resolved

File: app/migrations/Version20190116162854.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20190116162854 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE donations ADD nationality VARCHAR(2) DEFAULT \'FR\' NOT NULL');
+        $this->addSql('ALTER TABLE donations CHANGE nationality nationality VARCHAR(2) NOT NULL');
```
Comment (JeremieSamson) [inline app/migrations/Version20190116162854.php]: I don't get the second line, was it generated by a diff ?

Comment (ottaviano) [reply app/migrations/Version20190116162854.php]: the first one adds a new column with a default value for existing rows and the second one remove this default value

**Pull request merged**